### DISCUSSION TITLE
Error 500, internal server error.

### DISCUSSION THREAD

**Discussion by TECH-70** [vercel/community#279] [Help] [answered]
I have been developing a next js app, with tailwindcss, and when i run it locally, it works like a charm. No errors in the terminal at all. But when i deploy it to vercel, no matter what i try, vercel always gives me the error 500

**Comment by Pieparker** [comment, marked as answer]
Hey there! Your post is pretty vague however [this](https://www.reddit.com/r/nextjs/comments/j4i5c1/500_internal_server_error_on_vercel_deploy_how_to/) is likely to help you. In short, you should check your function logs to understand the error further.

>You can view your error logs by going to your Vercel dashboard, clicking on your project, and then clicking "View Function Logs". This will show errors happening in realtime. They are not persisted (they're streaming logs). Refresh the page, and you'll see your 500 error. That should tell you what the issue is.
